sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -4,6 +4,73 @@
 
 ```ts
 
+import { HttpOperationResponse } from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface GetSchemaByIdOptions extends OperationOptions {
+}
+
+// @public
+export interface GetSchemaIdOptions extends OperationOptions {
+}
+
+// @public
+export interface RegisterSchemaOptions extends OperationOptions {
+}
+
+// @public
+export interface Response {
+    _response: HttpOperationResponse;
```

Comment:
Is it common for this particular API that a developer would need to directly check headers or HTTP status code? If not, it might be worth omitting `Response` from types to reduce the number of interfaces here.

Reply:
It was my understanding that we always do have offer the _response as a pattern, is that not so? I didn't realize this was a call we'd make per API. :)

Reply:
Oh boy! 

I was under the assumption like Nick that we always have `_response` in the output which would be the raw response of the http request.

But it looks none of the 3 Keyvault packages or the Text Analytics one follow this pattern
App Config and the 3 storage packages do follow this pattern

I do believe there is a guideline around this across languages that there should be a way for users to get to the raw response.

@bterlson Do you recall why Key Vault does not follow the above pattern?

cc @sadasant, @jonathandturner

Reply:
Meanwhile, I would recommend inlining the `_response` in the `SchemaIdResponse` and `SchemaResponse` definitions instead of having a separate `Response` type being exported from the package. It is not of much use to the user outside of `SchemaIdResponse` and `SchemaResponse`

Reply:
I started on this in draft in #10800. I'm waiting for the discussion about the Java surface area tomorrow to further align to that.